code/modules/research/nanites/nanite_programs/healing.dm

```diff
@@ -176,29 +161,20 @@
 			return
 		var/update = FALSE
 		for(var/obj/item/bodypart/L in parts)
-			if(L.heal_damage(3/parts.len, 3/parts.len, null, BODYPART_ORGANIC))
+			if(L.heal_damage(2/parts.len, 2/parts.len, null, BODYPART_ORGANIC))
 				update = TRUE
 		if(update)
 			host_mob.update_damage_overlays()
 	else
-		host_mob.adjustBruteLoss(-3, TRUE)
-		host_mob.adjustFireLoss(-3, TRUE)
+		host_mob.adjustBruteLoss(-2, TRUE)
+		host_mob.adjustFireLoss(-2, TRUE)
 
 /datum/nanite_program/brain_heal_advanced
 	name = "Neural Reimaging"
-	desc = "The nanites are able to backup and restore the host's neural connections, potentially replacing entire chunks of missing or damaged brain matter."
+	desc = "The nanites are able to backup and restore the host's neural connections, potentially replacing entire chunks of missing or damaged brain matter. Consumes nanites even if it has no effect."
```

Comment:
There's no sensor that you can use to detect brain damage, though (nor is there one for foreign contaminants, come to think of it)... That kind of hurts this program's main usage as being a hard counter to hypnotic flashes.

Reply:
Kind of a bad idea to add one in, because then you just have the scenario of abductors mindcontrolling you... and then the mind control just immediately goes away.

Reply:
Abductor mind control doesn't use traumas to control you, so it wouldn't be affected by this.